subgraph/origin/schema.graphql

```diff
@@ -6,3 +6,38 @@ type AvailableStateRoot @entity {
   contractAddress: Bytes! # address
   uts: BigInt! # uint256
 }
+
+type DeclaredDepositIntent @entity {
+  id: ID!
+  tokenAddress: Bytes! # address
```

Comment:
Please align the ordering as per contract event DepositIntentDeclared

Reply:
Entities has been aligned as per contract events.